### DISCUSSION TITLE
best way to display simulation model ?

### DISCUSSION THREAD

**Discussion by PinkMushroom** [thliebig/openEMS-Project#61] [General]
I'm assuming that there is a way to do this, but I'm unable to find it.

I've looked through the CSXCAD python interface docs, and the only thing i see is Write2XML, but I have not found any sort of display function.

I did see in #48 that you can import an STL file, which is definitely useful :-) But i'm wondering what is the best way to view some of the geometries used in the examples since they are generated directly using CSXCAD calls.

Thank you.

**Comment by PinkMushroom** [comment]
And the answer is 

AppCSXCAD

be sure you have the openEMS/bin directory in your path, the python library does not put the full path in the module variable name, i.e. CSXCAD.AppCSXCAD_BIN is simply "AppCSXCAD".

Not sure how i missed that looking through the microstrip notch example.

**Reply by biergaizi** [reply]
The new documentation website needs still work.

The documentation from the old wiki has instructions on doing that:
https://wiki.openems.de/index.php/Tutorial:_First_Steps.html

**Reply by thliebig** [reply]
> The new documentation website needs still work.

Yes indeed, any help here would be welcome... the legacy wiki could be a good starting point to transfer infos, images and so on...
